web/angular-wallet/src/app/constants.ts

```diff
@@ -102,6 +102,15 @@ export const constants = {
       reliable: true,
       testnet: false
     },
+    {
+      name: 'Signum Node Latin America',
+      region: 'Middle America',
+      location: 'Dominican Republic',
```

Comment:
The node is actually in Miami,Florida (USA)
I picked this location because is commonly used by gamedevs who offers multiplayer services for the LATAM region